### PR TITLE
Feature/UI auth

### PR DESCRIPTION
**Pull request by Lex-23**
- create simple UI app on react
- add components: `login`, `auth-ping`, and `home`
- add `logout` function
- setup corsheaders for test for be
- add view and url for `auth-ping` to be

### CODE REVIEW

**Comment by Lex-23** [comment]
[Create UI auth part](https://trello.com/c/gg1W5R6d/56-create-ui-auth-part)

File: ui/cryptostock_app/src/axiosApi.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+});
+
+
+axiosInstance.interceptors.response.use(
+    response => response,
+    error => {
+        const originalRequest = error.config;
+
+        if (error.response.status === 401 && originalRequest.url === baseURL+'auth/refresh/') {
```
Comment (vadim-denisov) [inline ui/cryptostock_app/src/axiosApi.js:21]: I prefer format strings:
https://www.tutorialstonight.com/javascript-string-format.php#:~:text=Formatting%20string%20in%20javascript%20is,a%20dollar%20sign%20(%24).&text=These%20variables%20are%20called%20placeholders,the%20values%20of%20the%20variables.

File: ui/cryptostock_app/src/components/authPing.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            message:"",
+        };
+
+        this.getMessage = this.getMessage.bind(this)
+    }
+
+    async getMessage(){
```
Comment (vadim-denisov) [inline ui/cryptostock_app/src/components/authPing.js:14]: Why async?

Comment (Lex-23) [reply ui/cryptostock_app/src/components/authPing.js:14]: It's alternative variant for construction `.then`. I

File: ui/cryptostock_app/src/axiosApi.js (hunk 1 above)
Comment (Lex-23) [reply ui/cryptostock_app/src/axiosApi.js:21]: got it, looks good.

**Review by vadim-denisov — APPROVED** [review]

**Pull request merged**